apps/site/test/site_web/views/schedule_view_test.exs

```diff
@@ -435,4 +435,10 @@ defmodule SiteWeb.ScheduleViewTest do
              )
     end
   end
+
+  describe "is_station?" do
+    test "returns true if stop is station" do
+      assert is_station?(%Stop{id: "place-north", station?: true}) == true
```

Comment:
Let's add a test case for it returning false too. That's also part of what coverage measures! ;)